channel-manager-operations/reservations.md

```diff
@@ -40,21 +42,21 @@ This option allows creations, modifications, and partial or complete cancellatio
         "title": "Mister",
         "nationalityCode": "US",
         "languageCode": "en-US",
-        "telephone": "1-3526-88918"
+        "telephone": "1-3526-88918",
+        "loyaltyCode": "PG60972345",
+        "loyaltyInfo": {
+            "membershipId": "PG60972345",
+            "programCode": "BWR",
+            "tierCode": "Gold"
+        }
     },
     "sources": [
         {
-            "code": 1,
```

Comment:
Why yout have made it with single source and not kept is as multiple sources ?

Reply:
no reason in particular. When reservation is mews originated, it looks like this (1   mews source). I can revert it.